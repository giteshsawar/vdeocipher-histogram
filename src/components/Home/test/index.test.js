// Link.react.test.js
import React from "react";
import Homepage from "../index";
import { configure, shallow } from "enzyme";
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe("Home component", () => {

  it ("should render home component", () => {
    const component = shallow(<Homepage />);
    // console.log('shallow wrapper home \n\n', component);
    expect(component.getElements()).toMatchSnapshot();
  });
});
