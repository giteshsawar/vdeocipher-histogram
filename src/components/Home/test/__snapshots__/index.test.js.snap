// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render home component 1`] = `
Array [
  <div
    className="charts-container"
  >
    <DataSet
      dataset="pages"
      diff={100}
      max={2000}
    />
    <DataSet
      dataset="posts"
      diff={500}
      max={5000}
    />
  </div>,
]
`;

exports[`Link changes the class when hovered 1`] = `ShallowWrapper {}`;
